### PR TITLE
Consultancy Website as first Microverse Capstone Project

### PR DESCRIPTION
**Pull request by iAmWaris97**
- Personalized the content of the website design given.
- I chose my website to be for consultancy services.
- The pages look almost identical to the original design. Minor adjustments like text or image changes are made.
- I built only the home page and the about page.
- Each of these pages should have versions for 2 different screen sizes.
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.
- Some sections are created dynamically in JavaScript.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      </div>
+    </section>
+    <footer class="footer">
+      <p>
+        <img src="./assets/img/logo-sm.png" alt="">
+      </p>
+      <small
+        >(c) 2022 iAmWaris</small
+      >
+    </footer>
```
Comment (rotimiazeez) [inline about.html]: - Please note that on the mobile version Your footer seems a bit different from the original design, kindly fix this for a better UI 😄
Your design | Original design
screenshot (53) | screenshot (52)

File: about.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+          </div>
+        </div>
+        <div class="event">
+          <img src="/assets/img/winner-2.jpg" alt="Abuja conference" />
+          <div class="event-description">
+            <h3>2021</h3>
+            <p>ICCR Scholarship</p>
+          </div>
+        </div>
+      </div>
```
Comment (rotimiazeez) [inline about.html]: - The image on this page is broken, kindly fix it. 👍🏽
screenshot (55)
screenshot (54)

File: about.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+              <div class="mobile-menu">
+                <div class="menu-bar times">
+                  <i class="fa fa-times fa-1x"></i>
+                </div>
+                <ul class="nav-list">
+                  <li>
+                    <a href="/index.html" class="menu-bar" id="home-link"
+                      >Home</a
+                    >
+                  </li>
```
Comment (rotimiazeez) [inline about.html]: - Please note that when I try to navigate to the home from the about page I get a site not found error and vice-versa. Please fix this 😉
screenshot (56)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+            Our services covers assistance in availing Scholarships and fee
+            discounts, Assistance in VISA processing & Travel arrangements,
+            Assistance in tuition fee payment and refunds, Assistance in course
+            Selection, and assistance in applying to the universities around the
+            world.
+          </p>
+          <p class="event-date">2022.08.6(SAT) ~ 7(SUN)</p>
+          <small>@ iAmWaris Tower, Kabul.</small>
+        </div>
+      </main>
```
Comment (rotimiazeez) [inline index.html:116]: - Please make sure your design uses the same font, margin etc as seen in the original design
Original design | Your Design
screenshot (58) | screenshot (59)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente
+              quam vero sit commodi
+            </p>
+          </div>
+        </div>
+        <a href="#" class="program-desktop"> SEE ALL THE SERVICES</a>
+        <a href="#" class="program-mobile"> Join Upcoming Workshop 2022</a>
+      </div>
+    </section>
```
Comment (rotimiazeez) [inline index.html:173]: - For a better UI, kindly make sure your design matches that of the original design.
Your Design | Original Design
screenshot (61) | screenshot (60)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+    <section class="speakers">
+      <div class="speakers-title">
+        <h2>Key Personnel</h2>
+        <span></span>
+      </div>
+      <div class="keynote-speakers"></div>
+      <button type="button" class="btn" id="btn">
+        See More <span> <i class="fa fa-angle-down"></i></span>
+      </button>
+    </section>
```
Comment (rotimiazeez) [inline index.html:185]: - Please make sure your images are of the same sizes and the texts beside them are well aligned as seen in the original design.
- Original Design
screenshot (62)
screenshot (63)
- Your Design
screenshot (64)
screenshot (65)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @iAmWaris97  👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️
- Good README 📖 
- Good and descriptive PR 💯 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (iAmWaris97) [reply about.html]: Noted and solved

File: about.html (hunk 2 above)
Comment (iAmWaris97) [reply about.html]: Solved

File: about.html (hunk 3 above)
Comment (iAmWaris97) [reply about.html]: Solved with proper routing

File: index.html (hunk 4 above)
Comment (iAmWaris97) [reply index.html:116]: Font size increased

File: index.html (hunk 5 above)
Comment (iAmWaris97) [reply index.html:173]: Aligned and solved

File: index.html (hunk 6 above)
Comment (iAmWaris97) [reply index.html:185]: addressed issues were solved

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @iAmWaris97, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You have done a great job working on the requested changes✔️✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**